chapters/7-publication.tex

```diff
@@ -67,38 +66,39 @@ \subsection{Preparing manuscripts as {\LaTeX} documents}
 
 % what exactly is a dynamic document
 As we discussed in Chapter 6, the most widely used software
-for dynamically managing academic manuscripts is {\LaTeX} (pronounced ``lah-tek'').
+for dynamically managing academic manuscripts is {\LaTeX}.
 \index{\LaTeX}
-It requires explicitly encoded references to the latest versions of all outputs.
+{\LaTeX} uses explicit references to the file path of each input (e.g. tables and figures),
+which are reloaded from these locations every time the final document is compiled.
 This is not possible by default in Microsoft Word.
 There, you have to copy and paste each object
 whenever tables, graphs, or other inputs are updated.
-As time goes on, it becomes more and more likely
+As time goes on, it becomes increasingly likely
 that a mistake will be made or something will be missed.
 In {\LaTeX}, instead of writing in a
 ``what-you-see-is-what-you-get'' mode as you do in Word,
 you write plain text in a \texttt{.tex} file,
-interlaced with coded instructions for formatting and exhibits (similar to HTML).
+interlaced with coded instructions formatting the document and linking to exhibits (similar to HTML).
 {\LaTeX} manages tables and figures dynamically
 and includes commands for simple markup
 like font styles, paragraph formatting, section headers and the like.
 It includes special controls for
 footnotes and endnotes, mathematical notation, and bibliography preparation.
 It also allows publishers to apply global styles and templates to already-written material,
-allowing them to reformat entire documents in house styles with only a few keystrokes.
+reformatting entire documents in house styles with only a few keystrokes.
 
 % keeping it simple
 To create a {\LaTeX} manuscript,
-you will typically create a new top-level directory for each final output.
-Many projects go so far as to create documents like the main text
+you will typically start from a new top-level directory for each final output,
+such as a paper or report.
+Many projects go so far as to create the main paper
 and any appendix text as separate projects.
```

Comment:
I think cut or elaborate. As is, this is confusing to someone new to latex. If we want to keep we should explain or give examples to how that is done.